assets/js/components/ReportTable.js

```diff
@@ -24,8 +24,24 @@ import get from 'lodash/get';
 import invariant from 'invariant';
 import PropTypes from 'prop-types';
 
-export default function ReportTable( props ) {
-	const { rows, columns, className, limit, zeroState: ZeroState } = props;
+/**
+ * Internal dependencies.
+ */
+import { useFeature } from '../hooks/useFeature';
+import GatheringDataNotice from './GatheringDataNotice';
+
+export default function ReportTable( {
+	rows,
+	columns,
+	className,
+	limit,
+	zeroState: ZeroState,
+	isGatheringData = false,
```

Comment:
@jimmymadon @tofumatt I know the AC says to add the `isGatheringData` prop, but I think this should be `gatheringData` to be consistent with how we are doing it with other components. For e.g #4692 #4696

Reply:
Oh, shoot, you're right. Yeah, let's be consistent for now and use `gatheringData`. @jimmymadon pointed out it's weird we have both and maybe we should consolidate the naming in a future issue, but for now let's at least be consistent about where we're inconsistent 😅 

`gatheringData` it is 👍🏻

Reply:
Well, if we do consolidate it, I do prefer boolean variables to begin with is/has or a question verb (not sure if I read that within our coding conventions - but it might be a worth addition if not). So I would rather have this replaced to isGatheringData everywhere. But will change this to gatheringData for consistency now.